### PR TITLE
Update Decorators.md

### PR DESCRIPTION
**Pull request by cemremengu**
overwritable -> overridable

I added this as a review back then but I think I didnt submit it so just making sure

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Decorators.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Note: using an arrow function will break the binding of `this` to the Fastify `r
 <a name="decorators-encapsulation"></a>
 #### Decorators and encapsulation
 
-Decorators are not *overwritable* within the same encapsulated plugin. If you try to declare a decorator that was previously declared *(in other words, use the same name)*, `decorate`, `decorateRequest` and `decorateReply` will throw an exception.
+Decorators are not *overridable* within the same encapsulated plugin. If you try to declare a decorator that was previously declared *(in other words, use the same name)*, `decorate`, `decorateRequest` and `decorateReply` will throw an exception.
```
Comment (StarpTech) [inline docs/Decorators.md]: The whole sentence sounds really complicated. What about?

If you define a decorator with the same name more than once in the same plugin, fastify will throw an exception.

Comment (cemremengu) [reply docs/Decorators.md]: Much better indeed

Comment (StarpTech) [reply docs/Decorators.md]: The whole sentence would be:

> If you define a decorator (`decorate`, `decorateRequest` and `decorateReply`) with the same name more than once in the same plugin fastify will throw an exception.

Comment (cemremengu) [reply docs/Decorators.md]: In the same *encapsulated* plugin. Should we emphasise that?

Comment (StarpTech) [reply docs/Decorators.md]: Yes

**Pull request merged**